src/blockchain.js

```diff
@@ -1,6 +1,7 @@
-const SHA256 = require('crypto-js/sha256');
+const crypto = require('crypto');
```

Comment:
This is a great idea! However, do you know which versions of node support this? If so, you could add it to the README file as a requirement.

Reply:
i dont know where exactly it got introduced, but when i read the docs right, the functions we use here were in nodejs 1.0.0